src/shared/config.js

```diff
@@ -55,10 +55,11 @@ module.exports = [
   {
     name: 'weka',
     memeOfTheWeek: false,
-    standupCaptains: ['libbyschuknight', 'sandemchad', 'timothyjohn2015', 'fourseven', 'eileen', 'dave'],
+    standupCaptains: ['libbyschuknight', 'sandemchad', 'timothyjohn2015', 'fourseven', 'eileen', 'dave', 'nat', 'Charlotte' ],
```

Comment:
So we can call Natalie Nat? Is she ok with nick names?

Reply:
Also can we remove Chad from here please:
```suggestion
    standupCaptains: ['libbyschuknight', 'timothyjohn2015', 'fourseven', 'eileen', 'dave', 'nat', 'Charlotte' ],
```

Reply:
Oh thought I had removed Chad. Nat goes by Nat. See her slack handle, it is `@nat`. but am also double checking with her...

Reply:
Yep she is happy with Nat.

Reply:
I think she wants to change her name to Mat though 😂